app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,24 @@
+class InvitationsController < ApplicationController
+  before_filter :require_user
```

Comment:
`before_filter` should be `before_action` as filter will be deprecated in Rails 5.